Gateway/Request/AddressDataBuilder.php

```diff
@@ -94,17 +101,35 @@ public function build(array $buildSubject)
         }
         
         $shippingAddress = $order->getShippingAddress();
+
         if ($shippingAddress) {
-            
-            // filter housenumber from streetLine1
-            $requestDelivery = ["street" => $shippingAddress->getStreetLine1(),
-                "postalCode" => $shippingAddress->getPostcode(),
-                "city" => $shippingAddress->getCity(),
-                "houseNumberOrName" => '',
-                "stateOrProvince" => $shippingAddress->getRegionCode(),
-                "country" => $shippingAddress->getCountryId()
-            ];
 
+			if ($shippingAddress->getStreetLine1()) {
+				$address = $this->adyenHelper->getStreet($shippingAddress->getStreetLine1());
```

Comment:
adyenHelper->getStreet() wants an address as a parameter, it does not work with string